gammapy/image/tests/test_core.py

```diff
@@ -261,6 +261,56 @@ def test_cutout_paste_wcs_error(self):
         with pytest.raises(WcsError):
             image.paste(cutout)
 
+    @pytest.mark.parametrize(('shape', 'factor', 'proj'), [((4, 4), 2, 'CAR'),
```

Comment:
I'd suggest to always use asymmetric shapes for tests.
Then flipped axis index errors are noticed.